pkg/scheduler/framework/interface.go

```diff
@@ -179,9 +179,9 @@ type Status struct {
 	code    Code
 	reasons []string
 	err     error
-	// failedPlugin is an optional field that records the plugin name a Pod failed by.
+	// abortPlugin is an optional field that records the plugin name a scheduling cycle is aborted by.
 	// It's set by the framework when code is Unschedulable, UnschedulableAndUnresolvable or Pending.
-	failedPlugin string
+	abortPlugin string
```

Comment:
This is not an acceptable name https://inclusivenaming.org/word-lists/tier-1/abort/

Can we just say `plugin` here? It just means that this is the plugin that caused this status code.

But if you want to keep the idea that this is specific about "failures", maybe `rejecterPlugin`? https://en.wiktionary.org/wiki/rejecter

Reply:
Oops, OK just learned it.

> Can we just say plugin here? It just means that this is the plugin that caused this status code.

Yes, makes sense. Let's go for just `plugin` as it's simple.